Library/Formula/tgui.rb

```diff
@@ -0,0 +1,29 @@
+class Tgui < Formula
+  desc "GUI library for use with sfml"
+  homepage "http://tgui.eu"
+  url "https://github.com/texus/TGUI/archive/v0.7-alpha2.tar.gz"
```

Comment:
If this isn't stable it needs to live in https://github.com/Homebrew/homebrew-devel-only